packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -140,6 +141,7 @@ const LocaleModalCountries = ({
             region.countries.map((country, index) => (
               <a
                 key={index}
+                tabIndex="0"
```

Comment:
question(non-blocking): Does this mean we should make this `<a>` sequential-focusable even if `country.href` is missing? Or does this have another intent?

Reply:
Thanks @asudoh, this isn't needed. Good call 👍